src/js/pages/JobsPage.js

```diff
@@ -0,0 +1,66 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {RouteHandler} from 'react-router';
+
+import Page from '../components/Page';
+import SidebarActions from '../events/SidebarActions';
+import TabsMixin from '../mixins/TabsMixin';
+
+class JobsPage extends mixin(TabsMixin) {
+  constructor() {
+    super();
```

Comment:
@rcorral Following that argument, we should always call the `super` whenever we override a method, as we otherwise miss something, if one of the classes in the chain evolves.
Let's please add this topic to our agenda and discuss it, as I see value in doing so, but on the other hand it kinda seems like overhead.